app/js/components/manageApps/ManageApps.jsx

```diff
@@ -329,10 +329,9 @@ export default class ManageApps extends React.Component {
                 showMsg: true,
                 msgBody: `${fileName} has been successfully installed`,
                 msgType: "success",
-                appList: resultData,
-                staticAppList: resultData,
               };
```

Comment:
What exactly was appList and staticAppList doing and did it affect the functionality when removed

Reply:
`resultData` was an array representing a list of owas returned by the server as part of the callback once upload of an owa is successful. This is what was used to render the list of `owas` after upload. But since what we wanted is the list of both `owas` and `modules` ,we already had a method that does this, and as such i just called it.